### PR TITLE
Doc: add test case diagram

### PR DESCRIPTION
**Pull request by squirrelsc**

### CODE REVIEW

**Comment by LiliDeng** [comment]
do we need add before_case and after_case introduction?

**Comment by squirrelsc** [comment]
> do we need add before_case and after_case introduction?

I will add some

File: docs/write_test/write_case.rst [hunk 1]
```diff
@@ -52,23 +52,25 @@ can save everyone’s time.
...
 
-The LISA test is composed of :ref:`write_test/write_case:metadata`,
-:ref:`write_test/write_case:test case body` and
-:ref:`write_test/write_case:setup and clean-up`.
+A typical test case includes definition, setup, run, and validation.
 
-Metadata
-~~~~~~~~
+   .. figure:: ../img/sample.png
+      :alt: sample test case
```
Comment (dsrivastavv) [inline docs/write_test/write_case.rst:58]: Should we add `cleanup` step in the diagram as well?

Comment (squirrelsc) [reply docs/write_test/write_case.rst:58]: I will add some descriptions in after_case. It needs to keep the sample simple, so it's easy to follow.

**Review by dsrivastavv — APPROVED** [review]

**Review by LiliDeng — APPROVED** [review]

**Pull request merged**